contracts/mocks/ExampleERC20.sol

```diff
@@ -30,4 +31,11 @@ contract ExampleERC20 is ERC20Burnable {
 
         _mint(msg.sender, amount);
     }
+
+    function mint(address account, uint256 amount) external {
+        // Can only mint 10 at a time.
```

Comment:
Is this comment correct? If _MAX_MINT is `1e16` that would mean decimals is `15`?